docs/pages/management/export-audit-events/datadog.mdx

```diff
@@ -0,0 +1,417 @@
+that defines a `teleport-event-handler` role and a user with read-only access
+to the `event` API:
+
+```yaml
+kind: user
+metadata:
+  name: teleport-event-handler
+spec:
+  roles: ['teleport-event-handler']
+version: v2
+---
+kind: role
+metadata:
+  name: teleport-event-handler
+spec:
+  allow:
+    rules:
+      - resources: ['event']
+        verbs: ['list','read']
+version: v5
+```
+
+Use `tctl` to create the role and the user:
+
+```code
+$ tctl create -f teleport-event-handler-role.yaml
+# user "teleport-event-handler" has been created
+# role 'teleport-event-handler' has been created
+```
+
+## Step 4/6. Create teleport-event-handler credentials
+
+### Enable impersonation of the Event Handler user
+
+In order for the Teleport Event Handler plugin to forward events from your Teleport
+cluster, it needs a signed identity file from the cluster's certificate authority.
+The `teleport-event-handler` user cannot request this itself, and requires another
+user to **impersonate** this account in order to request a certificate.
+
+Create a role that enables your user to impersonate the Fluentd user. First,
+paste the following YAML document into a file called
+`teleport-event-handler-impersonator.yaml`:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: teleport-event-handler-impersonator
+spec:
+  # SSH options used for user sessions
+  options:
+    # max_session_ttl defines the TTL (time to live) of SSH certificates
+    # issued to the users with this role.
+    max_session_ttl: 10h
+
+  # allow section declares a list of resource/verb combinations that are
+  # allowed for the users of this role. by default nothing is allowed.
+  allow:
+    impersonate:
+      users: ["teleport-event-handler"]
+      roles: ["teleport-event-handler"]
+```
+
+Next, create the role:
+
+```code
+$ tctl create -f teleport-event-handler-impersonator.yaml
+```
+
+(!docs/pages/includes/add-role-to-user.mdx role="teleport-event-handler-impersonator"!)
+
+### Export an identity file for the Event Handler plugin user
+
+The Teleport Event Handler plugin uses the `teleport-event-handler` role and user to
+read events. We export an identity file for the user with the `tctl auth sign`
+command.
+
+<Tabs>
+<TabItem label="Self-Hosted" scope={["oss","enterprise"]}>
+```code
+$ tctl auth sign --user=teleport-event-handler --out=identity
+```
+
+This command creates one PEM-encoded file, `identity`. The identity file
+includes both TLS and SSH credentials. The Event Handler plugin uses the SSH
+credentials to connect to the Proxy Service, which establishes a reverse tunnel
+connection to the Auth Service. The plugin uses this reverse tunnel, along with
+your TLS credentials, to connect to the Auth Service's gRPC endpoint.
+
+</TabItem>
+<TabItem label="Teleport Cloud" scope={["cloud"]}>
+```code
+$ tctl auth sign --user=teleport-event-handler --out=identity
+```
+
+This command creates one PEM-encoded file, `identity`. The identity file
```

Comment:
Looks like this entire block is duplicated in both tabs - would it make sense to move it out of the tabs so we don't risk forgetting to update both sections in tandem in the future?

Reply:
Yes, but can you think of a good title for the consolidated tab label, to identify the tab as referring to Self-Hosted or Cloud, but not Helm", to contrast the "Helm Chart" tab?

Reply:
Hmm. Nothing comes to mind. This is a tricky one. I've already approved though, so if we don't think of a good option feel free to merge as-is.

Reply:
$5 says @ptgott has got a great idea when he comes around to reviewing this one.

Reply:
Excellent! Now, who do I collect my $5 from?

Reply:
Resolved in a72c62d1ffe6995a82a7dda8a8ee8cfadd846751